apps/src/sharedComponents/footer/CopyrightDialog/index.tsx

```diff
@@ -10,6 +10,12 @@ import i18n from '@cdo/locale';
 import './style.scss';
 import defaultStyle from '@cdo/apps/sharedComponents/accessible-dialogue.module.scss';
 
+// External code can specify additional content to be shown, or clear it again.
```

Comment:
Should we mention that this is currently only used by Music Lab to show image attribution information?

Reply:
Good call; added.